bot.go

```diff
@@ -651,11 +651,194 @@ func (b *Bot) GetFile(fileID string) (File, error) {
 	return responseRecieved.Result, nil
 }
 
+// Use this method for your bot to leave a group, supergroup or channel. Returns True on success.
+func (b *Bot) LeaveChat(recipient Recipient) (bool, error) {
```

Comment:
This is bot API , I just do like documentations , But you right , I will change it